src/main/java/com/st/lms/utils/DateCalculations.java

```diff
@@ -0,0 +1,72 @@
+package com.st.lms.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateCalculations {
+
+	//gets today's date in yyyy-MM-dd
+	public static Date getCurrentTime() {

```

Comment:
Since callers rely on the fact that these methods return objects which are `java.sql.Date` instances, they should be declared as returning `java.sql.Date`.